meson.build

```diff
@@ -0,0 +1,16 @@
+project('imgui', 'cpp',
+    version: '1.76',
+    license: 'MIT'
+)
+
+include_dirs = include_directories('.', 'examples')
```

Comment:
Why do you includes `examples`? I am not familiar with the project, in general this sounds strange.
If this is intentional I would appreciate a comment why headers from the `examples` directory are supposed to be included into the other projects.

Reply:
Imgui needs to link to a backend API to load openGL functions. It provides a lot of premade headers with the more "popular" ones, and they are located in `examples`. I will add a comment.

Reply:
Thank you.